test/acceptance/.snyk

```diff
@@ -1,10 +0,0 @@
-# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
-version: v1.25.0
-# ignores vulnerabilities until expiry date; change duration by modifying expiry date
-ignore:
-  SNYK-JS-INFLIGHT-6095116:
-    - '*':
-        reason: None Given
-        expires: 2024-09-17T14:55:27.170Z
```

Comment:
Makes me think whether we should be ignoring vulnerabilities this way of via the UI.

Reply:
Yeah that is an option and keeps it under version control